client/src/app/project/project.component.ts

```diff
@@ -38,19 +38,51 @@ export class ProjectComponent implements OnInit {
   processesDataSource: MetadataDataSource<MetadataDocument>;
   filesDataSource: MetadataDataSource<MetadataDocument>;
 
+  sharedTabConfig = [
+    {
+      key: 'project',
+      label: '1. Project'
```

Comment:
I think these texts are duplicated in `project-registration-form.component.html`

Reply:
Will fix in [#225](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/225)